modules/ROOT/pages/configuration/cypher-version-configuration.adoc

```diff
@@ -0,0 +1,37 @@
+[[cypher-version-configuration]]
+= Configure the Cypher default version
+:page-role: new-2025.06
+:description: How to configure the Cypher default version.
+
+You can specify the version of Cypher® in which you want to run your queries, choosing between Cypher 5 and Cypher 25.
+Cypher 5 is the default version for all newly created databases, however, if you want to take advantage of the new features in Cypher 25, you can set the default version to Cypher 25.
+For more information, see xref:introduction.adoc#_cypher_versions[Cypher® versions].
+
+To specify the Cypher version, use one of the following options:
+
+Configure a default Cypher version for the whole DBMS::
+The default language version for the whole DBMS can be configured in the _neo4j.conf_ file using the setting xref:configuration/configuration-settings.adoc#config_db.query.default_language[db.query.default_language] (default value: CYPHER_5). +
+Changing this setting in an existing DBMS *does not* affect existing databases.
+It only applies to newly created databases unless the version is specified as part of the `CREATE` or `ALTER` database commands.
+
+Configure a default Cypher version per database::
+The default language for a specific database can be set using a Cypher database administration command with the `SET DEFAULT LANGUAGE` clause when creating the database or by altering the database after it has been created.
+This clause determines the default Cypher version for the specified database, overriding the version set in the configuration file.
+For example, `CREATE DATABASE mydb SET DEFAULT LANGUAGE CYPHER 25` or `ALTER DATABASE mydb SET DEFAULT LANGUAGE CYPHER 5`.
+For more details and examples, see xref:database-administration/syntax.adoc#administration-syntax-database-management[Database management command syntax] and the respective pages in the xref:database-administration/index.adoc[] section. +
+If not specified, the default language for the database is set to the default language of the DBMS.
+Setting the default language requires the `SET DATABASE DEFAULT LANGUAGE` privilege.
```

Comment:
Would this be correct to say?
```suggestion
Setting the default language using `ALTER DATABASE` requires the `SET DATABASE DEFAULT LANGUAGE` privilege.
```

Reply:
yes and no, it would be correct for standard databases but not composite databases (both uses ALTER DATABASE)
If you want to be lazy we could just not specify what the privileges are:
```suggestion
Setting the default language requires the correct privileges.
```
or we could be more explicit:
```suggestion
Modifying the default language for a standard database requires the `SET DATABASE DEFAULT LANGUAGE` privilege and for a composite database the `ALTER COMPOSITE DATABASE` privilege.
Setting the default language on creation only require the correct `CREATE` privilege.
```